lib/broadside/gli/commands.rb

```diff
@@ -14,7 +14,7 @@ def add_instance_flag(cmd)
   cmd.desc '0-based index into the array of running instances'
   cmd.default_value 0
   cmd.arg_name 'INSTANCE'
-  cmd.flag [:n, :instance], type: Fixnum
+  cmd.flag [:n, :instance], type: Integer
```

Comment:
interesting http://blog.bigbinary.com/2016/11/18/ruby-2-4-unifies-fixnum-and-bignum-into-integer.html